library/globals.inc.php

```diff
@@ -2028,6 +2028,17 @@ function gblTimeZones()
     //
     'Notifications' => array(
 
+        'oefax_enable' => array(
+            xl('Enable Fax SMS Module'),
+            array(
+                0 => xl('Disabled'),
+                1 => xl('RingCentral'),
+                2 => xl('Twilio'),
+            ),
+            '0',
+            xl('Enable Fax SMS Support. Remember to install fax module.')
+        ),
```

Comment:
I am pretty sure Ken added a event listener for globals. Would likely need @adunsulag 's or @kchapple 's input on that.

Reply:
If it's there I can get it done. I need to get this in 'cause I still have to bring back to 502 with another billing fix Ray spotted.

Reply:
github won't let me attach files right now, but here's a snippet of how to add globals to your module so you don't have to put unnecessary code in the core. You add your lestener to the GlobalsInitialized event and then add your globals using the functions.

`public function onBootstrap(MvcEvent $e)
    {
        $serviceManager = $e->getApplication()->getServiceManager();
        $oemrDispatcher = $serviceManager->get(EventDispatcherInterface::class);
        $oemrDispatcher->addListener(GlobalsInitializedEvent::EVENT_HANDLE, [$this, 'addCustomGlobals']);
    }

    public function addCustomGlobals(GlobalsInitializedEvent $event) {

        $event->getGlobalsService()->createSection("Fetch Files", "Connectors");

        $setting = new GlobalSetting( "Fetch Files Enabled", 'bool', 1, "When checked, run fetch, when unchecked, pause" );
        $event->getGlobalsService()->appendToSection( "Fetch Files", "ff_fetch_enabled", $setting );

        $setting = new GlobalSetting( "Local Storage Dir", 'text', "/var/www/htdocs/openemr/sites/default/documents/fetch_files", "Where to put fetched files, requires an absolute path on server." );
        $event->getGlobalsService()->appendToSection( "Fetch Files", "ff_local_dir", $setting );

        $setting = new GlobalSetting( "Max Files", 'text', 10, "Number ofiles to process in a batch." );
        $event->getGlobalsService()->appendToSection( "Fetch Files", "ff_max_files", $setting );
}`

Reply:
Thanks @kchapple i'll have a look.